main.go

```
@@ -319,19 +319,8 @@ func brrr(userid string, text []string) string {
 	if err != nil {
 		return fmt.Sprintf("ERROR: %s (%s)", err.Error(), userid)
 	}
-	emoji := strings.TrimSpace(text[1])
+	emoji, _ := parseEmoji(text[1])
```

feel like we should use the bool returned to throw an error earlier rather than waiting to try minting an empty emoji.

---

updated. wdyt?